service/common/src/main/java/org/apache/polaris/service/admin/PolarisAdminService.java

```diff
@@ -116,7 +117,7 @@ public class PolarisAdminService {
   private final SecurityContext securityContext;
   private final AuthenticatedPolarisPrincipal authenticatedPrincipal;
   private final PolarisAuthorizer authorizer;
-  private final PolarisMetaStoreManager metaStoreManager;
+  private final PolarisDaoManager daoManager;
```

Comment:
Using this class to show case how business logic invokes type-specific DAO objects. We can easily extend it to other classes like `BasePolarisCatalog`, `PolarisEntityManager` and `Resolver`